docs/mixins/state.md

```diff
@@ -0,0 +1,179 @@
+# Marionette.Toolkit.StateMixin
+
+`Marionette.Toolkit.StateMixin` is a public mixin for [`App`](../app.md) or any Marionette class definition that uses a `Backbone.Model` for keeping state.
+
+Use a `Marionette.Toolkit.StateMixin` if your object/view needs to maintain information that isn't business data.  This provides a consistent method for storing and getting state, along with triggering related events.
+
+## Documentation Index
+
+* [Using Marionette.Toolkit.StateMixin](#using-marionette.toolkit.stateMixin)
```

Comment:
I think just StateMixin is ok here and in most places.  Compare to old StateClass docs for its usage of the full Marionette.Toolkit.StateClass